### DISCUSSION TITLE
Q about PurgeInfoCache API Method

### DISCUSSION THREAD

**Discussion by kefo** [cantaloupe-project/cantaloupe#497] [Q&A]
We've noticed that Cantaloupe retains a source file's info json file in memory (though I don't know for how long).  

[We discovered this after deleting the info json file from disk manually and yet Cantaloupe continued to use the old/same info json file.  Restarting Cantaloupe, which purged the memory, forced Cantaloupe to recognize that a new info json file needed to be created.]

Does the PurgeInfoCache remove all the info json files from the on-disk cache or from Cantaloupe's in-memory info json cache or from both?

Related question:  How long does Cantaloupe retain a info json file in memory.

**Comment by adolski** [comment]
I believe that this is all answered here: https://cantaloupe-project.github.io/manual/5.0/caching.html#Info%20Cache
